packages/documentation/.storybook/main.ts

```diff
@@ -90,13 +85,27 @@ const config: StorybookConfig = {
     STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_FALLBACK: 'dev',
     STORYBOOK_BASE_URL: 'https://design-system.post.ch',
   }),
+
   async viteFinal(config: InlineConfig) {
     return mergeConfig(config, {
       css: {
         devSourcemap: true,
       },
     });
   },
+
+  features: {
+    actions: false,
+    backgrounds: false,
+    highlight: false,
+    outline: false,
+    measure: false,
+    viewport: false,
```

Comment:
We should try this addon, it was broken in the previous version but would be useful in my opinion!

Reply:
Done.